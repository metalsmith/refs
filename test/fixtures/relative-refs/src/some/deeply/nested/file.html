---
refs:
  about: ../../../about/index.html
---