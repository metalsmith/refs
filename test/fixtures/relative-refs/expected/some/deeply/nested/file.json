{
  "refs": {
    "about": {
      "id": "about/index.html"
    }
  },
  "id": "some/deeply/nested/file.html"
}